src/InvoiceBundle/Action/Create.php

```diff
@@ -41,8 +41,12 @@ public function __construct(FormHandler $handler, ClientRepository $clientReposi
 
     public function __invoke(Request $request, Client $client = null)
     {
-        if (!$this->clientRepository->getTotalClients()) {
+        $totalClientsCount = $this->clientRepository->getTotalClients();
+        if (!$totalClientsCount) {
             return new Template('@SolidInvoiceInvoice/Default/empty_clients.html.twig');
+        } elseif (1 === $totalClientsCount && is_null($client)){
```

Comment:
* The `elseif` can be removed since the previous `if` already returns (So this can just change to an `if` bellow)
* Please use `null === $client` instead if `is_null`
* Missing space between the closing `)` and opening `{`